misc/scripts/download.sh

```diff
@@ -31,8 +31,10 @@ if ! wget -q --tries=10 --timeout=20 --spider https://github.com; then
 fi
 
 if curl --output /dev/null --silent --head --fail "$URL" ; then
-	mkdir -p "$HOME/.cache/pacstall" && cd "$HOME"/.cache/pacstall/
-	mkdir -p "$PACKAGE" && cd "$PACKAGE"
+	if [[ "$type" = "install" ]]; then
+		mkdir -p "/tmp/pacstall/pacscripts/" && cd "/tmp/pacstall/pacscripts/"
+		mkdir -p "$PACKAGE" && cd "$PACKAGE"
+	fi
```

Comment:
Needs an `else` for when `$type == "download` to specify where to download the pacscript

Reply:
@Henryws says when `-D` flag is active, then just download it in `pwd`, so not specifying an `else` will automatically download it in `pwd`

Reply:
Ok